search/includes/classes/class-search.php

```diff
@@ -2465,16 +2465,22 @@ public function get_cached_custom_results_existence() {
 	 * @return bool $exist If there are any Custom Search Results or not.
 	 */
 	public function is_custom_results_exist() {
+		remove_filter( 'ep_enable_do_weighting', [ $this, 'filter__ep_enable_do_weighting' ], 9999, 4 );
```

Comment:
@rinatkhaziev @rebeccahum `remove_filter` accepts 3 arguments so the 4th one can be safely removed.

Reply:
Thanks for the catch, this has been addressed in #3794